src/js/calculator.js

```diff
@@ -0,0 +1,78 @@
+import {MESSAGE} from "./constant.js";
```

Comment:
인덴팅도 신경쓰면 더 좋을 것 같아요 :) 
```suggestion
import { MESSAGE } from "./constant.js";
```